tools/smoother_benchmarking/README.md

```diff
@@ -0,0 +1,99 @@
+# Planners Smoothing Benchmark
+
+This experiment runs a set with randomly generated goals for objective benchmarking.
+
+Bechmarking scripts require the following python packages to be installed:
+
+```
+pip install transforms3d
+pip install seaborn
+pip install tabulate
+```
+
+For the benchmarking purposes, the clarification of execution time should be made for planner and smoother servers:
```

Comment:
Move this stuff to be last, after the config file modifications

Reply:
Done in https://github.com/ros-planning/navigation2/pull/3236/commits/5b8c3a23baf9fa68b7ff38c18dbaf5b07fac8bab